packages/compass-indexes/src/components/create-index-form/mdb-code-viewer.tsx

```diff
@@ -0,0 +1,64 @@
+import { Code, Link, css, spacing } from '@mongodb-js/compass-components';
+import React from 'react';
+
+const containerStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+});
+
+const programmingLanguageLinkStyles = css({
+  marginLeft: 'auto',
+  marginTop: spacing[100],
+});
+
+const MDBCodeViewer = ({
+  dbName,
+  collectionName,
+  indexNameTypeMap,
+  dataTestId,
+}: {
+  dbName: string;
+  collectionName: string;
+  indexNameTypeMap: { [key: string]: string };
+  dataTestId?: string;
+}) => {
+  const generateCode = () => {
+    let codeStr = `db.getSiblingDB("${dbName}").getCollection("${collectionName}").createIndex{(\n`;
+
+    Object.entries(indexNameTypeMap).forEach(([name, type], index) => {
+      // Replacing everything inside the parenthesis i.e. (asc)
+      const parsedType = type.replace(/\(.*?\)/g, '').trim();
+      codeStr += `  "${name}": "${parsedType}"`;
```

Comment:
Another edge case to consider is that field names can have quotes in them. We should escape those `"` characters (similar to collection names).